src/FileInput/FileInput.less

```diff
@@ -0,0 +1,37 @@
+@import (reference) "../less/index";
+
+.FileInput {
+  .padding--x--s;
+  .text--medium;
+  .border--s(@neutral_silver);
+  position: relative;
+  color: @neutral_dark_gray;
+  background: @neutral_white;
+  box-sizing: border-box;
+  height: @size_4xl;
+  width: 100%;
+  cursor: pointer;
+  .text--caps;
+
+  .FileInput--Text {
+    .alignSelf(center);
+
+    &.selected {
```

Comment:
Not correctly named. Should be FileInput--Text--selected

See http://clever.github.io/components/#/less/less-style-guide for naming conventions.